source/material_model/diffusion_dislocation.cc

```diff
@@ -362,13 +362,25 @@ namespace aspect
           prm.declare_entry ("Minimum strain rate", "1.4e-20", Patterns::Double(0),
                              "Stabilizes strain dependent viscosity. Units: $1 / s$");
           prm.declare_entry ("Minimum viscosity", "1e17", Patterns::Double(0),
-                             "Lower cutoff for effective viscosity. Units: $Pa s$");
+                             "Lower cutoff for effective viscosity. Units: $Pa \\, s$");
           prm.declare_entry ("Maximum viscosity", "1e28", Patterns::Double(0),
-                             "Upper cutoff for effective viscosity. Units: $Pa s$");
+                             "Upper cutoff for effective viscosity. Units: $Pa \\, s$");
           prm.declare_entry ("Effective viscosity coefficient", "1.0", Patterns::Double(0),
                              "Scaling coefficient for effective viscosity.");
           prm.declare_entry ("Reference viscosity", "1e22", Patterns::Double(0),
-                             "Reference viscosity for nondimensionalization. Units $Pa s$");
+                             "The reference viscosity that is used for pressure scaling. "
+                             "To understand how pressure scaling works, take a look at "
+                             "\\cite{KHB12}. In particular, the value of this parameter "
+                             "would not affect the solution computed by \\aspect{} if "
+                             "we could do arithmetic exactly; however, computers do "
+                             "arithmetic in finite precision, and consequently we need to "
```

Comment:
I would prefer we put this large blob into the manual only and then put a reference here instead of duplicating it in every material model.

Reply:
I tried to find a place where this would fit, but we don't discuss the pressure scaling anywhere in the manual. In fact, we don't discuss any of our numerical methods in the manual at all, and on purpose -- that's what the papers are there for.

So I agree with you, but I couldn't figure out how to do that without writing a section I didn't want to write and that I wouldn't know where to put.

Reply:
I am fine either way, but if you decide to put this in the manual, I think the end of section "2.4 Static or dynamic pressure?" could be a good place.